dist/availity-angular-ui.js

```diff
@@ -628,6 +640,15 @@
           avValField.validate(ngModel.$viewValue);
         });
 
+        // Removes all errors on page, does not reset view or model values, this is to be handled by the app
+        scope.$on(AV_VAL.EVENTS.RESET, function () {
+          var violations = ngModel.avResults.violations;
+          ngModel.avResults.violations = [];
+          element.parents(AV_BOOTSTRAP_ADAPTER.CLASSES.FORM_GROUP).removeClass('has-error');
```

Comment:
The original intent for the validation framework was to decouple the validation for a field element from the UI rendering of a validation error.  This couples them together.